python/packages/autogen-agentchat/src/autogen_agentchat/teams/_utils.py

```diff
@@ -0,0 +1,89 @@
+from datetime import datetime
+import json
+import logging
+import sys
+from typing import Optional, Union, List, Dict, Any, Sequence
+from dataclasses import asdict, is_dataclass
+
+from .group_chat._events import ContentPublishEvent
+from ..agents import ChatMessage, TextMessage, MultiModalMessage, ToolCallMessage, ToolCallResultMessage, StopMessage
+from autogen_core.components import FunctionCall, Image
+from autogen_core.components.models import FunctionExecutionResult
+
+ContentType = Union[str, List[Union[str, Image]],
+                    List[FunctionCall], List[FunctionExecutionResult]]
+
+
+class AgentChatLogHandler(logging.Handler):
```

Comment:
Naming. Could use a more informative name like `ConsoleLogHandler` and `FileLogHandler`

Reply:
Agreed. Done.